src/main/java/com/flightstats/hub/metrics/ChannelType.java

```diff
@@ -0,0 +1,7 @@
+package com.flightstats.hub.metrics;
+
+public enum ChannelType {
```

Comment:
not 100% sure, but I think we generally use SCREAMING_SNAKE_CASE for enum values, since they're kinda like constants.

Reply:
good point, changed this.